cli/src/main/java/org/apache/falcon/cli/FalconInstanceCLI.java

```diff
@@ -22,81 +22,118 @@
 import org.apache.commons.cli.Option;
 import org.apache.commons.cli.OptionGroup;
 import org.apache.commons.cli.Options;
-import org.apache.falcon.FalconCLIConstants;
+import org.apache.falcon.client.FalconCLIConstants;
+import org.apache.falcon.ValidationUtil;
 import org.apache.falcon.LifeCycle;
 import org.apache.falcon.ResponseHelper;
 import org.apache.falcon.client.FalconCLIException;
 import org.apache.falcon.client.FalconClient;
 import org.apache.falcon.resource.InstanceDependencyResult;
 
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
+import static org.apache.falcon.client.FalconCLIConstants.RUNNING_OPT;
```

Comment:
These lengthy imports make me feel to modify the checkstyle rule of disallowing * imports